safehttp/doc.go

```diff
@@ -123,6 +123,25 @@
 //
 // TODO
 //
+// Incremental Adoption
+//
+// In order to migrate your service using http.Handlers to the safehttp package,
+// we recommend you start doing that one endpoint at a time. Use the
+// ServeMux.RegisteredHandler method to do this. See:
+//
+//  safeMuxConfig := /* configured ServeMuxConfig, including interceptors */
+//  safeMuxConfig.Handle("/bar", safehttp.MethodGET, barSafeHandler)
```

Comment:
Maybe add a line
`safeMuxConfig.Handle("/bar", safehttp.MethodPOST, barSafeHandler)`
to showcase method "overloading". It wasn't clear to me immediately that this would work as expected when we call `RegisteredHandler` later.

Reply:
Done.